lib/rubocop/cop/rails/find_by.rb

```diff
@@ -23,6 +23,7 @@ class FindBy < Cop
 
         def on_send(node)
           return unless where_first?(node)
+          return if where_with_method_as_param?(node)
```

Comment:
@Drenmi @pocke @bbatsov  I should change naming here, because now we check for method call and variable. We need to consider string as well. When I know all requirements I'll refactor this bit as well.